app/components/maps/MapHeader.tsx

```diff
@@ -1,48 +1,40 @@
-import React, { useEffect, useState } from "react";
+import React from "react";
+import { useRouter } from "next/router";
 import { PageHeader, Button } from "antd";
 
-import { getTmxId } from "../../lib/api/tmxRequests";
+import { MapInfo } from "../../lib/api/apiRequests";
+import { cleanTMFormatting } from "../../lib/utils/formatting";
 
 interface Props {
-    mapUId: string;
+    mapInfo: MapInfo;
 }
 
-export const MapHeader = ({ mapUId }: Props): JSX.Element => {
-    const [tmxId, setTmxId] = useState("");
-
-    useEffect(() => {
-        const fetchTmxId = async () => {
-            const fetchedTmxId = await getTmxId(mapUId);
-            if (fetchedTmxId) {
-                setTmxId(fetchedTmxId);
-            }
-        };
-        fetchTmxId();
-    }, []);
+export const MapHeader = ({ mapInfo }: Props): JSX.Element => {
+    const router = useRouter();
 
     return (
         <PageHeader
-            onBack={() => null} // TODO: add link to home when that exists
+            onBack={() => router.push(`${location.origin}/`)}
```

Comment:
Is there a reason to prefix `location.origin` here? Using just "/" seems to work fine too

Reply:
Ye, makes perfect sense - I think I played around with the router before this and had to use origin for that.